salesagility/SuiteCRM
# Feature/gdpr fixes 2 green ticks only

## Description




- PR for single 'opt-in' settings: shows only a green tick on list-view if opt-in (or nothing)
- this pr also fixing a bug on admin / password settings page.

## Motivation and Context


## How To Test This

part 1:
 - set dropdown to 'opt-in' on Email Settings
 - person list views should shows only a green tick if opt-in or nothing
part 2: (password bug fix)
 - switch on each password security option on password settings page
 - try to change your password => see validation message about the wrong password at top of the page
 - try switch off each pwd security settings => save button should works

## Types of changes


### Final checklist
